zql/docs/processors/README.md

```diff
@@ -154,6 +155,58 @@ ssl   1521912240.189735 CSbGJs3jOeB6glWLJj 10.47.7.154 27137     52.85.83.215 44
 
 ---
 
+## `fuse`
+
+|                           |                                                   |
+| ------------------------- | ------------------------------------------------- |
+| **Description**           | Return results under a single record definition that unifies the field and type information across all records in the query result. |
```

Comment:
I think "return" is the wrong terminology for a dataflow pipeline.  How about "Transforms input records to output records that unify... "

Reply:
@mccanne: Fair point. Your suggestion looks good to me, so I've committed that change.